src/javascript/app/base/page.js

```diff
@@ -92,10 +95,14 @@ const Page = (() => {
                 checkLanguage();
                 RealityCheck.onLoad();
                 Menu.init();
+                LocalStore.remove('date_first_contact');
```

Comment:
I'm not sure if we need it here, if we're doing the same in socket_general. what do you think?

Reply:
You are absolutely right. removing it now.